x/perp/client/cli/tx.go

```diff
@@ -29,12 +29,70 @@ func GetTxCmd() *cobra.Command {
 		AddMarginCmd(),
 		OpenPositionCmd(),
 		ClosePositionCmd(),
+		MultiLiquidateCmd(),
 		DonateToEcosystemFundCmd(),
 	)
 
 	return txCmd
 }
 
+func MultiLiquidateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "multi-liquidate ubtc:unusd:nibi1zaavvzxez0elundtn32qnk9lkm8kmcsz44g7xl ueth:unusd:nibi1zaavvzxez0elundtn32qnk9lkm8kmcsz44g7xl",
```

Comment:
I think this should be changed in better way.
```suggestion
		Use:   "multi-liquidate [Pair1:Trader1] [Pair2:Trader2] ...",
```